src/Websites/Websites/help/Set-AzWebApp.md

```diff
@@ -561,7 +561,15 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ### Microsoft.Azure.Commands.WebApps.Models.PSSite
 
 ## NOTES
+Below provided cmdlet will help you to update Azure Web App to **DOTNETCORE**
 
+$PropertiesObject = @{
+    "CURRENT_STACK" =  "dotnetcore"
+}
+New-AzResource -PropertyObject $PropertiesObject -ResourceGroupName "Default-Web-WestUS" -ResourceType Microsoft.Web/sites/config -ResourceName "ContosoWebApp/metadata" -ApiVersion 2018-02-01 -Force
```

Comment:
I'm curious to know why the `Force` parameter is being used? It's preferable to omit the `Force` parameter in examples unless it's required. This is due to confirmations being suppressed which could cause a customer to inadvertently overwrite something in their production environment without realizing it.

Reply:
yes agree with you, we can remove this Force Parameter, I kept that one because sometimes if you already created Azure Webapp with .NET runtime then it doesn't use to get change to .NETCORE until we force them. There is already an issue been raised and Microsoft suggested to use in that way and even I tested so placed Force Parameter

[github Issue Link](https://github.com/MicrosoftDocs/azure-docs/issues/44432)